homeassistant/core.py

```diff
@@ -219,6 +219,7 @@ def async_add_job(self, target: Callable[..., None], *args: Any) -> None:
         # if a task is sheduled
         if task is not None:
             self._pending_tasks.add(task)
+            return task
```

Comment:
We should not return tasks, if people want to await a task, they should just use `yield from`